core/src/main/java/org/jdbi/v3/core/argument/BuiltInArgumentFactory.java

```diff
@@ -53,4 +54,16 @@
             .flatMap(factory -> JdbiOptionals.stream(factory.build(expectedType, value, config)))
             .findFirst();
     }
+
+    private static class EnumByNameArgumentFactory implements ArgumentFactory {
```

Comment:
this is purely legacy support, `QualifiedEnumArgumentFactory`'s structure doesn't really allow extracting just this `ColumnMapper`